packages/codemirror-graphql/package.json

```
@@ -14,26 +14,30 @@
   },
   "license": "MIT",
   "files": [
+    "dist",
     "hint.js",
     "info.js",
     "jump.js",
     "lint.js",
     "mode.js",
+    "*.flow",
     "utils",
     "variables",
     "results",
     "README.md",
     "LICENSE"
   ],
+  "main": "dist",
   "options": {
     "mocha": "--full-trace --require resources/mochaBootload src/*/__tests__/*-test.js",
     "mocha_tdd": "--full-trace --watch --require resources/mochaBootload src/**/*/__tests__/*-test.js"
   },
   "scripts": {
     "lint": "eslint src",
     "check": "flow check",
-    "build": "babel src --root-mode upward --ignore __tests__ --out-dir .",
-    "build-js": "node ../../resources/buildJs.js",
+    "build": "yarn build-clean && yarn build-js && yarn build-flow .",
+    "build-js": "babel src --root-mode upward --ignore src/__tests__ --out-dir .",
+    "build-clean": "rimraf {mode,hint,info,jump,lint}.{js,js.flow} && rimraf esm results utils variables coverage __tests__",
```

I _think_ we should wrap this in quotes to avoid shell expansion (so rimraf does the expansion instead, in a more cross-platform way):

```suggestion
    "build-clean": "rimraf '{mode,hint,info,jump,lint}.{js,js.flow}' && rimraf esm results utils variables coverage __tests__",
```